source/common/network/address_impl.cc

```diff
@@ -43,6 +43,13 @@ void validateIpv6Supported(const std::string& address) {
   }
 }
 
+// Construsts a readable string with the embedded nulls in the abstract path replaced with '@'.
+std::string friendlyNameFromAbstractPath(absl::string_view path) {
+  std::string friendly_name(path.data(), path.size());
```

Comment:
Use absl::StrReplaceAll? then you don't have to explicitly making a copy here.

Reply:
absl::StrRepalceAll doesn't play well with replacing single null characters because it grabs the replacement by a string_view without the size parameter:
https://github.com/abseil/abseil-cpp/blob/19b021cb3ff23048dfbe236a4e611925d8930831/absl/strings/str_replace.h#L164

I think at this point even if it looks messy, I could pass in a reference to `friendly_name` and modify it in the function with `std::replace`.

Reply:
If you make StrToStrMap typed as string_view (or std::string) instead of `const char *`, then it can play, but not a big deal here.